src/app/containers/InlineContainer/index.jsx

```diff
@@ -0,0 +1,26 @@
+import React, { useContext } from 'react';
+import { ServiceContext } from '../../contexts/ServiceContext';
+import Blocks from '../Blocks';
+import fragment from '../Fragment';
+import InlineLink from '../InlineLink';
+import InlineSpan from '../../components/InlineSpan';
+import inlineBlockPropTypes from '../../models/propTypes/inline';
```

Comment:
Are these up to date?

https://github.com/bbc/simorgh/blob/26154fc29befe062bdace2e2e77532d7ad679df1/src/app/models/propTypes/inlineLink/index.js#L5-L14

Seeing this error in console

```
Failed prop type: The prop `isRequired` is marked as required in `InlineContainer`, but its value is `undefined`.
```